bgpd/bgp_rpki.c

```diff
@@ -620,6 +622,14 @@ static int start(void)
 		rtr_mgr_free(rtr_config);
 		return ERROR;
 	}
+
+	/* Wait till at least one group is fully synchronized
+	 * with the server.
+	 * https://rtrlib.readthedocs.io/en/latest/development.html
+	 */
+	while (!rtr_mgr_conf_in_sync(rtr_config))
+		sleep(1);
```

Comment:
A sleep is going to be a no from me.  There is absolutely no guarantee that we would ever come out of this.  We need to find a better way to do this that can get the answer back and continue on while allowing the rest of bgp to continue working